Lib/multiprocessing/shared_memory.py

```diff
@@ -74,6 +74,8 @@ class SharedMemory:
     def __init__(self, name=None, create=False, size=0):
         if not size >= 0:
             raise ValueError("'size' must be a positive integer")
+        if create and size == 0:
+            raise ValueError("'size' of new segment must be greater than 0")
         if create:
```

Comment:
I hate to nit-pick, but I think it would be easier to read this if the "size == 0" check were moved after line 79 (the "if create" test. But it's not a big deal.

Reply:
Sure, done.